snapshot/manifest.go

```diff
@@ -28,6 +28,8 @@ type Manifest struct {
 	RootEntry *DirEntry `json:"rootEntry"`
 
 	RetentionReasons []string `json:"-"`
+
+	Tags map[string]string `json:"tags,omitempty"`
```

Comment:
@jkowalski Q: does it make sense to store the tags here as well (besides the manifest labels)? This is needed mainly to facilitate displaying them when listing snapshots.
Also, if the `tag:` prefix is added to the labels, then it makes sense to strip it from the tags here. Thoughts?

Reply:
Yes, I would store them in the snapshot manifest (for display purposes) and create corresponding labels when saving snapshots for indexing purposes.